core/src/main/scala/akka/persistence/r2dbc/internal/postgres/PostgresSnapshotDao.scala

```diff
@@ -231,4 +272,84 @@ private[r2dbc] class PostgresSnapshotDao(settings: R2dbcSettings, connectionFact
     }
   }.map(_ => ())(ExecutionContexts.parasitic)
 
+  override def currentDbTimestamp(): Future[Instant] = {
```

Comment:
added a bunch of stuff to the SnapshotDao to be able to use the existing BySlice abstraction